### ISSUE TITLE
(bug) The video plays every time I respond to a call on my phone.

### ISSUE THREAD

**Issue by sNrVJx** [TeamNewPipe/NewPipe#1722]
If I have a video paused and I answer a call, the video plays automatically. Is this a bug?

**Comment by snappyapple632** [comment]
No, that's the "Resume on focus gain" feature. When you exit out of the app and open it again, it will start right where you left off. It's useful if you find yourself locking and unlocking your phone often like I do.
